modules/tinymce/src/plugins/table/main/ts/ui/UiUtils.ts

```diff
@@ -17,10 +17,15 @@ interface Item {
 const onSetupToggle = (editor: Editor, formatName: string, formatValue: string) => {
   return (api: Toolbar.ToolbarMenuButtonInstanceApi) => {
     const boundCallback = Singleton.unbindable();
+    const isNone = Strings.isEmpty(formatValue);
 
     const init = () => {
-      api.setActive(editor.formatter.match(formatName, { value: formatValue }));
-      const binding = editor.formatter.formatChanged(formatName, api.setActive);
+      // If value is empty (A None-entry in the list), check it only if the format is not set at all. Otherwise, check it if the format is set to the correct value.
+      const setActive = (matched) =>
+        api.setActive(isNone ? !matched : matched);
+
+      setActive(editor.formatter.match(formatName, { value: formatValue }, undefined, isNone));
+      const binding = editor.formatter.formatChanged(formatName, setActive, isNone);
```

Comment:
Can you please add a note about this highlighting all items and add a TODO to TINY-7713?